io.liri.BaseApp.yaml

```diff
@@ -0,0 +1,52 @@
+id: io.liri.BaseApp
+branch: stable
```

Comment:
`stable` is a really bad name for baseapps since they depend upon the runtime they use they need to be versioned. 

If this requires the KDE runtime that sadly means you have to version it as `5.11` and make a new branch every 6 months.

Reply:
Done thanks.